docs/GETTING_STARTED.md

```diff
@@ -34,14 +34,16 @@ If repair history exists for a table, ecChronos will use the last time repair wa
 On the other hand, if there's no repair history present for a table, the ecChronos will assume that the table has been repaired in the past.
 
 The assumption is done in the following way:
+* Initial delay = must be lower than Repair interval, configurable via ecc.yml
 * Repair delay in days = min(repair interval, 1)
```

Comment:
This is only correct if the default of 1 day is kept. If the user state another delay, it isn't a correct assumption).

Reply:
Repair delay is initial delay now. This is therefore not resolved and the information is not correct due to the introduction with this fix.